### ISSUE TITLE
404 errors

### ISSUE THREAD

**Issue by dschrempf** [haskell/cabal-website#2]
I am not sure if this is the right place to report this issue, but I do not know, where it would fit. It is stated that ../release/ is not part of this repository but it is not stated which repository ../release/ is part of.

I keep running into 404 errors on the haskell documentation homepages.

The latest one is related to cabal. I am trying to find documentation about the language extension `FlexibleInstances` but I simpy do not succeed...

On

    https://www.haskell.org/cabal/release/cabal-latest/doc/API/Cabal/Language-Haskell-Extension.html

under

    data KnownExtension

basically all links are broken, this includes

    http://www.haskell.org/ghc/docs/latest/html/users_guide/type-class-extensions.html#instance-rules

I think it is really one of the major problems of Haskell, that documentation cannot be found and is hidden in endless cascades of links from one page to another. Sorry for letting this out on you here.

Thank you very much,
Dominik

**Comment by 23Skidoo** [comment]
This was fixed recently, see https://github.com/haskell/cabal/issues/4582 and http://haskell.github.io/cabal-website/doc/html/Cabal/Language-Haskell-Extension.html.
